zebra-network/src/peer/handshake.rs

```diff
@@ -25,20 +25,20 @@ use crate::{
     BoxedStdError, Config,
 };
 
-use super::{error::ErrorSlot, server::ServerState, HandshakeError, PeerClient, PeerServer};
+use super::{Client, ErrorSlot, HandshakeError, Server};
 
 /// A [`Service`] that handshakes with a remote peer and constructs a
 /// client/server pair.
-pub struct PeerHandshake<S> {
+pub struct Handshake<S> {
     config: Config,
     internal_service: S,
     timestamp_collector: mpsc::Sender<MetaAddr>,
     nonces: Arc<Mutex<HashSet<Nonce>>>,
 }
 
-impl<S: Clone> Clone for PeerHandshake<S> {
+impl<S: Clone> Clone for Handshake<S> {
     fn clone(&self) -> Self {
-        Self {
+        Handshake {
```

Comment:
Why isn't this kept as `Self`?

Reply:
I'm not sure exactly, but I used VS Code's RLS integration to do symbol renaming, so I guess it looked up all symbols of type `PeerHandshake` and renamed them, and the inner `Self` matched but the outer `Self` didn't?